### PR TITLE
fix wizard behaviour when detecting appropriate action

### PR DESCRIPTION
**Pull request by buixor**
- properly warn user when using `--binupgrade` or `--upgrade` in wrong context

### CODE REVIEW

File: scripts/test_ip_management.sh [hunk 1]
```diff
@@ -260,6 +260,20 @@ function test_ipv6_ip
     docurl ${APIK} "/v1/decisions?range=1111:2222:3333:4444:5555:6666:7777:8888/64&&contains=false" | ${JQ} '.[].value == "1111:2222:3333:4444:5555:6666:7777:8888"' > /dev/null || fail
     bouncer_echo "getting decisions for ip/range in 1111:2222:3333:4444:5555:6666:7777:8888/64"
 
+    cscli_echo "adding decision for ip 1111:2222:3333:4444:5555:6666:7777:8889"
+    ${CSCLI} decisions add -i 1111:2222:3333:4444:5555:6666:7777:8889 > /dev/null 2>&1
+
+    cscli_echo "deleting decision for ip 1111:2222:3333:4444:5555:6666:7777:8889"
+    ${CSCLI} decisions delete -i 1111:2222:3333:4444:5555:6666:7777:8889
```
Comment (buixor) [inline scripts/test_ip_management.sh:267]: don't you want to control the number of delete entries here ?

File: scripts/test_ip_management.sh [hunk 2]
```diff
@@ -260,6 +260,20 @@ function test_ipv6_ip
...
+
+    ${CSCLI} decisions list -i 1111:2222:3333:4444:5555:6666:7777:8889 -o json | ${JQ} '. == null' > /dev/null || fail
+    cscli_echo "getting decisions for ip 1111:2222:3333:4444:5555:6666:7777:8889 after delete"
+
+    cscli_echo "deleting decision for range 1111:2222:3333:4444:5555:6666:7777:8888/64"
+    ${CSCLI} decisions delete -r 1111:2222:3333:4444:5555:6666:7777:8888/64 --contained
```
Comment (buixor) [inline scripts/test_ip_management.sh:273]: same here

File: scripts/test_ip_management.sh [hunk 3]
```diff
@@ -273,10 +287,10 @@ function test_ipv6_range
     cscli_echo "adding decision for range aaaa:2222:3333:4444::/64"
     ${CSCLI} decisions add -r aaaa:2222:3333:4444::/64 > /dev/null 2>&1 || fail
      
-    ${CSCLI} decisions list -o json | ${JQ} '.[0].decisions[0].value == "aaaa:2222:3333:4444::/64", .[1].decisions[0].value == "1111:2222:3333:4444:5555:6666:7777:8888"' > /dev/null || fail
+    ${CSCLI} decisions list -o json | ${JQ} '.[0].decisions[0].value == "aaaa:2222:3333:4444::/64"' > /dev/null || fail
```
Comment (buixor) [inline scripts/test_ip_management.sh:290]: why not checking the output here ?

File: scripts/test_ip_management.sh [hunk 4]
```diff
@@ -352,6 +366,23 @@ function test_ipv6_range
     docurl ${APIK} "/v1/decisions?ip=bbbb:db8:0000:0000:0000:8fff:ffff:ffff" | ${JQ} '. == null' > /dev/null || fail
     bouncer_echo "getting decisions for ip in bbbb:db8:0000:0000:0000:8fff:ffff:ffff"
 
+    cscli_echo "deleting decision for range aaaa:2222:3333:4444:5555:6666:7777:8888/48"
+    ${CSCLI} decisions delete -r aaaa:2222:3333:4444:5555:6666:7777:8888/48 --contained > /dev/null 2>&1 || fail
```
Comment (buixor) [inline scripts/test_ip_management.sh:370]: why not checking the output here ?

File: scripts/test_ip_management.sh [hunk 5]
```diff
@@ -352,6 +366,23 @@ function test_ipv6_range
...
+    ${CSCLI} decisions list -o json -r aaaa:2222:3333:4444::/64 | ${JQ} '. == null' > /dev/null || fail
+    cscli_echo "getting decisions for range aaaa:2222:3333:4444::/64 after delete"
+
+    cscli_echo "adding decision for ip bbbb:db8:0000:0000:0000:8fff:ffff:ffff"
+    ${CSCLI} decisions add -i bbbb:db8:0000:0000:0000:8fff:ffff:ffff > /dev/null 2>&1 || fail
+    cscli_echo "adding decision for ip bbbb:db8:0000:0000:0000:6fff:ffff:ffff"
+    ${CSCLI} decisions add -i bbbb:db8:0000:0000:0000:6fff:ffff:ffff > /dev/null 2>&1 || fail
+
+    cscli_echo "deleting decision for range bbbb:db8::/81"
+    ${CSCLI} decisions delete -r bbbb:db8::/81 --contained > /dev/null 2>&1 || fail
```
Comment (buixor) [inline scripts/test_ip_management.sh:381]: why not checking the output here ?

**Review by buixor — COMMENTED** [review]
suggested changes

**Review by AlteredCoder — APPROVED** [review]

**Pull request merged**